src/Service.java

```diff
@@ -0,0 +1,13 @@
+public interface Service {
+    public class ServiceStation implements Service {
+        public void check(Transport transport) {
+
+        }
+    }
+}
```

Comment:
Этот интерфейс будут реализовывать транспортные средства. Не ServiceStation.
Поэтому тут достаточно написать вот так:
```suggestion
public interface Service {
        void check();
}
```

Reply:
Обрати также внимание, что модификатор доступа тут не нужен